packages/core/src/routes/authn.ts

```diff
@@ -16,15 +16,39 @@ export default function authnRoutes<T extends AnonymousRouter>(router: T) {
   router.get(
     '/authn/hasura',
     koaGuard({
-      query: z.object({ resource: z.string().min(1) }),
+      query: z.object({ resource: z.string().min(1), unauthorizedRole: z.string().optional() }),
```

Comment:
I think this should be `unauthenticatedRole` instead

Reply:
hmmm, I read the Hasura document again, and they indeed use auauthorized-role... I'm a bit confused